src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -11530,6 +11531,165 @@ at Microsoft.CodeAnalysis.Diagnostics.AnalyzerManager.AnalyzerExecutionContext.<
         }
 #endif
 
+        private static ImmutableArray<byte> CreateCSharpAnalyzerNetStandard13(string analyzerAssemblyName)
```

Comment:
note: this is not new code, I just moved it from the core test utilities project to here.